src/me/ryanhamshire/GriefPrevention/Claim.java

```diff
@@ -913,7 +913,7 @@ long getPlayerInvestmentScore()
 						{
 							score += 10;
 						}
-						else if(creativeMode && (block.getType() == Material.LAVA || block.getType() == Material.STATIONARY_LAVA))
+						else if(creativeMode && (block.getType() == Material.LAVA || block.getType() == Material.FLOWING_LAVA))
```

Comment:
Is stationary lava now flowing lava? Or did you just make this change... why?

Reply:
LAVA = stationary
FLOWING_LAVA = moving

-> there are a lot of name changes to make it better such as REDSTONE_DIODE_ON -> REPEATER etc.

I tested compiling it, works fine :)

Reply:
Ok so why did you change this to check for flowing instead of stationary...?

Reply:
Uhm it firstly checked for flowing lava or stationary lava and now it ischecking for exact the same but in a different order (stationary || flowing)

Reply:
Oh I see, LAVA is now referring to stationary.